subjects/concatalternate/README.md

```diff
@@ -0,0 +1,46 @@
+## concatalternate
+
+### Instructions
+
+Write a function `ConcatAlternate()` that receives two slices of an `int` as arguments and returns a new slice with the result of the alternated values of each slice.
+- The input slices can be of different lengths.
+- The new slice should return the elements of the larger slice first and then the elements of the smaller slice.
```

Comment:
fix: "The new slice should start with an element of the largest slice." - clearer this way

Reply:
It is fixed. Thank you